drf_braces/serializers/form_serializer.py

```diff
@@ -286,6 +300,7 @@ def validate(self, data):
                 raise serializers.ValidationError(form.errors)
 
             else:
+                self.capture_failed_fields(data, form.errors)
```

Comment:
from consistency standpoint other hooks accept an exception argument whereas this accepts dict of errors. makes sense to be consistent across different hooks?

Reply:
The other hooks are in the following format: field name, field data, validation exception, because we want to capture the field that had the error, the data that causes the error, and the error message. In this case `data` contains both information that we need and `form.errors` contains the fields that have the errors and the errors themselves. I don't see a need for having a separate parameter for an exception, but if you do, I'll add it.